src/stories/src/components/split-view/split-view.mdx

```diff
@@ -0,0 +1,265 @@
+import {
+  Default as SplitViewDemo,
+  LeadingPanel as LeadingDemo,
+  MultiplePanelsSplit as MultipleSplitDemo,
+  MultiplePanelsStacked as MultipleStackedDemo
+} from './split-view.stories.tsx';
+import { SplitViewDefaultCodeHtml } from './code/split-view-default.ts';
+import { SplitViewLeadingCodeHtml } from './code/split-view-leading.ts';
+import { SplitViewVerticalCodeHtml } from './code/split-view-vertical.ts';
+import { SplitViewMultipleSplitCodeHtml } from './code/split-view-multiple-split.ts';
+import { SplitViewMultipleStackedCodeHtml } from './code/split-view-multiple-stacked.ts';
+
+<PageSection>
+
+# Split view
+
+The split view component presents `<forge-split-view-panel>` elements side-by-side as resizable panels.
+
+</PageSection>
+
+---
+
+<PageSection>
+
+## Example
+
+### Default
+
+<LiveDemo layout="column" fullWidth codeHtml={SplitViewDefaultCodeHtml()}>
+  <SplitViewDemo />
+</LiveDemo>
+
+### Leading panel
```

Comment:
Same as adding more context for the Default demo, let's describe what `position` is doing here so it's more clear.

Same for the rest of the examples while we're at it. It will just help clear things up before questions are asked.